code/modules/mob/living/simple_mob/subtypes/animal/alien animals/catslug.dm

```diff
@@ -278,3 +278,112 @@
 	. = ..()
 	color = held.color
+
+/datum/category_item/catalogue/fauna/catslug/spaceslug
+	name = "Alien Wildlife - Catslug - Miros"
+	desc = "This catslug serves as the Fuel Depots resident attendant,\
+	despite the facility being fully automated and self-serve. - \
+	The Catslug is an omnivorous terrestrial creature.\
+	Exhibiting properties of both a cat and a slug (hence its name)\
+	it moves somewhat awkwardly. However, the unique qualities of\
+	its body make it exceedingly flexible and smooth, allowing it to\
+	wiggle into and move effectively in even extremely tight spaces.\
+	Additionally, it has surprisingly capable hands, and moves quite\
+	well on two legs or four. Caution is advised when interacting\
+	with these creatures, they are quite intelligent, and proficient\
+	tool users."
+	value = CATALOGUER_REWARD_MEDIUM	//Should offer a measure of incentive for people to visit the depot more often.
+
+/mob/living/simple_mob/vore/alienanimals/catslug/spaceslug
+	name = "Miros"
+	desc = "Looks like catslugs can into space after all! This little chap seems to have gotten their mitts on a tiny spacesuit, there's a nametag on it that reads \"Miros\" alongside the Aether Atmospherics logo."
+	tt_desc = "Mollusca Felis Stellaris"
+	icon_state = "spaceslug"
+	icon_living = "spaceslug"
+	icon_rest = "spaceslug_rest"
+	icon_dead = "spaceslug_dead"
+	digestable = 0
+	catalogue_data = list(/datum/category_item/catalogue/fauna/catslug/spaceslug)
+	holder_type = /obj/item/weapon/holder/catslug/spaceslug
+	makes_dirt = 0
+	say_list_type = /datum/say_list/catslug/spaceslug
+	
+	minbodytemp = 0				// Shamelessly stolen temp & atmos tolerances from the space cat.
+	maxbodytemp = 900			
+	heat_damage_per_tick = 3	
+	cold_damage_per_tick = 2
+	
+	min_oxy = 0
+	max_oxy = 0
+	min_tox = 0
+	max_tox = 0
+	min_co2 = 0
+	max_co2 = 0
+	min_n2 = 0
+	max_n2 = 0
+	
+	player_msg = "You are an intelligent creature capable of more than most think, clad in a spacesuit that protects you from the ravages of vacuum and hostile atmospheres alike. You can pick up and use many things, and even carry some of them with you into the vents, which you can use to move around quickly. You're quiet and capable, you speak with your hands and your deeds! <br>- - - - -<br> <span class='notice'>Keep in mind, your goal should generally be to survive. You're expected to follow the same rules as everyone else, so don't go self antagging without permission from the staff team, but you are able and capable of defending yourself from those who would attack you for no reason.</span>"
+	
+	has_langs = list("Sign Language")
+
+/datum/say_list/catslug/spaceslug
+	speak = list("Have any porl?", "What is that?", "What kind of ship is that?", "What are you doing?", "How did you get here?", "Don't take off your helmet.", "SPAAAAAACE!", "WAOW!", "Nice weather we're having, isn't it?")
+
+/mob/living/simple_mob/vore/alienanimals/catslug/spaceslug/Initialize()
+	. = ..()
```

Comment:
Did you test and see if this catslug gets the recolor verb, because I'm pretty sure that this callback will give it to them. 

It might be better to do a 'verbs -= /mob/living/simple_mob/vore/alienanimals/catslug/proc/catslug_color'

Or maybe it won't matter one way or the other, since it seems unlikely that people would play as them all that often~

Reply:
Added!